clas/corporate.md

```diff
@@ -0,0 +1,92 @@
+
+2. Grant of Copyright License. Subject to the terms and conditions of this 
+Agreement, You hereby grant to deps.cloud and to recipients of software 
+distributed by deps.cloud a perpetual, worldwide, non-exclusive, no-charge, 
+royalty-free, irrevocable copyright license to reproduce, prepare derivative 
+works of, publicly display, publicly perform, sublicense, and distribute Your 
+Contributions and such derivative works.
+
+3. Grant of Patent License. Subject to the terms and conditions of this 
+Agreement, You hereby grant to deps.cloud and to recipients of software 
+distributed by deps.cloud a perpetual, worldwide, non-exclusive, no-charge, 
+royalty-free, irrevocable (except as stated in this section) patent license 
+to make, have made, use, offer to sell, sell, import, and otherwise transfer 
+the Work, where such license applies only to those patent claims licensable 
+by You that are necessarily infringed by Your Contribution(s) alone or by 
+combination of Your Contribution(s) with the Work to which such 
+Contribution(s) was submitted. If any entity institutes patent litigation 
+against You or any other entity (including a cross-claim or counterclaim in 
+a lawsuit) alleging that your Contribution, or the Work to which you have 
+contributed, constitutes direct or contributory patent infringement, then any 
+patent licenses granted to that entity under this Agreement for that 
+Contribution or Work shall terminate as of the date such litigation is filed.
+
+4. You represent that You are legally entitled to grant the above license. You 
+represent further that each employee of the Corporation designated by You is 
+authorized to submit Contributions on behalf of the Corporation.
+
+5. You represent that each of Your Contributions is Your original creation 
+(see section 7 for submissions on behalf of others).
+
+6. You are not expected to provide support for Your Contributions, except to 
+the extent You desire to provide support. You may provide support for free, 
+for a fee, or not at all. Unless required by applicable law or agreed to in 
+writing, You provide Your Contributions on an "AS IS" BASIS, WITHOUT WARRANTIES 
+OR CONDITIONS OF ANY KIND, either express or implied, including, without 
+limitation, any warranties or conditions of TITLE, NON-INFRINGEMENT, 
+MERCHANTABILITY, or FITNESS FOR A PARTICULAR PURPOSE.
+
+7. Should You wish to submit work that is not Your original creation, You may 
+submit it to deps.cloud separately from any Contribution, identifying the 
+complete details of its source and of any license or other restriction 
+(including, but not limited to, related patents, trademarks, and license 
+agreements) of which you are personally aware, and conspicuously marking the 
+work as "Submitted on behalf of a third-party: [named here]".
+
+8. It is your responsibility to notify deps.cloud when any change is required 
+to the list of designated employees authorized to submit Contributions on 
```

Comment:
Since you're not collecting a list of designated employees, and asking the company to sign on behalf of all employees, this section should be amended to remove that section.